src/Sulu/Bundle/AdminBundle/Resources/js/components/Dialog/dialog.scss

```diff
@@ -33,6 +34,7 @@ $transitionDuration: 300ms;
     transition: opacity $transitionDuration;
     z-index: 1;
     pointer-events: none;
+    max-height: calc(100vh - $dialogMarginVertical);
```

Comment:
Why isn't that variable called `dialogMarginBottom`, to be similar to the `Overlay` component?

Reply:
The `Overlay` has no `margin-top` because it is positioned at the top of the screen. The `Dialog` is centered, therefore calling it `dialogMarginBottom` feels wrong 🙂